### PR TITLE
initial release

### PR DESCRIPTION
**Pull request by GilbN**
logo https://github.com/linuxserver/docker-templates/pull/253

### CODE REVIEW

File: root/etc/cont-init.d/50-config [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# create folders
+mkdir -p /config/www/xbackbone
+
+# copy xbackbone files
+if [ ! -e "/config/www/xbackbone/index.php" ]; then
+    cp -R /app/xbackbone/* /config/www/xbackbone
+    rm -rf /app/xbackbone
```
Comment (thespad) [inline root/etc/cont-init.d/50-config]: This feels a bit pointless as it'll only do it on first run and removing the files doesn't really achieve anything.

File: root/etc/cont-init.d/50-config [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+fi
+
+# permissions
+chown -R abc:abc \
+    /config
```
Comment (thespad) [inline root/etc/cont-init.d/50-config]: Missing a trailing newline

File: root/etc/cont-init.d/50-config (hunk 1 above)
Comment (GilbN) [reply root/etc/cont-init.d/50-config]: Thanks. Fixed

File: readme-vars.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  - { arch: "{{ arch_armhf }}", tag: "arm32v7-latest"}
+
+# container parameters
+param_container_name: "{{ project_name }}"
+param_usage_include_vols: true
+param_volumes:
+  - { vol_path: "/config", vol_host_path: "<path to data on host>", desc: "config directory volume mapping" }
+param_usage_include_env: true
+param_env_vars:
+  - { env_var: "TZ", env_value: "<TZ>", desc: "Timezone (i.e., America/New_York)" }
```
Comment (Roxedus) [inline readme-vars.yml]: Remove <> from the values, provide working examples.

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: readme-vars.yml (hunk 3 above)
Comment (GilbN) [reply readme-vars.yml]: done

**Review by Roxedus — APPROVED** [review]

**Pull request merged**